packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -23,7 +23,14 @@ module.exports = (resolve, rootDir, isEjecting) => {
     : undefined;
 
   const config = {
-    collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}', '!src/**/*.d.ts'],
+    collectCoverageFrom:[
+      "src/**/*.{js,jsx,ts,tsx}",
+      "!src/**/*.d.ts",
+      "!src/**/*.stories.{js,jsx,ts,tsx}",
+      "!src/serviceWorker.js",
+      "!src/setupProxy.js",
+      "!src/locales/*"
+    ],
```

Comment:
Let's add `src/index.js` as well. I don't see a good way of testing that right now, so let's leave it out until we have a good way to do that and it makes sense

Reply:
done and done